buildSrc/src/main/kotlin/ort-kotlin-conventions.gradle.kts

```diff
@@ -111,8 +111,9 @@ detekt {
 }
 
 val javaVersion = JavaVersion.current()
-val maxKotlinJvmTarget = runCatching { JvmTarget.fromTarget(javaVersion.majorVersion) }
+val jvmSourceCompatibility =  runCatching { JvmTarget.fromTarget(javaVersion.majorVersion) }
     .getOrDefault(enumValues<JvmTarget>().max())
+val jvmTargetCompatibility = JvmTarget.JVM_11
```

Comment:
@sschuberth Does it make sense to keep the logic to determine the max version and use different versions for source and target or should we just hard code both to Java 11?

Reply:
@sschuberth Any chance we can get this change into tomorrow's release?

Reply:
That's my goal, yes, and I'm on it. Just learned a few unexpected things about Gradle toolchains that require me to think about it a bit more.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/8411, which I propose to use instead of this PR.